src/components/common-components/Header.vue

```diff
@@ -98,7 +98,8 @@ export default defineComponent({
   computed: {
     ...mapGetters({
       userdata: 'getUser',
-      orders: 'getListOfOrders'
+      orders: 'getListOfOrders',
+      user_company: 'getUserCompany',
```

Comment:
User company is already fetched in userdata.

Reply:
changed